datalake/src/main/java/com/sequenceiq/datalake/entity/operation/SdxOperationStatus.java

```diff
@@ -0,0 +1,92 @@
+package com.sequenceiq.datalake.entity.operation;
+
+import java.util.UUID;
+
+import javax.persistence.Convert;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+import javax.persistence.UniqueConstraint;
+import javax.validation.constraints.NotNull;
+
+import com.sequenceiq.datalake.converter.SdxOperationTypeEnumConverter;
+import com.sequenceiq.datalake.converter.SdxDatabaseDrStatusTypeEnumConverter;
+
+@Entity
+@Table(uniqueConstraints = @UniqueConstraint(columnNames = {"operationId"}))
+public class SdxOperationStatus {
```

Comment:
I think `SdxOperation` would fit better, there is too much status around this class